packages/web/src/pages/UserInterface/index.tsx

```diff
@@ -83,7 +102,13 @@ export default function UserInterface(): React.ReactElement {
           {!configLoading && (
             <Form
               onSubmit={handleUserInterfaceUpdate}
-              defaultValues={nestObject<UserInterface>(config)}
+              defaultValues={
+                nestObject<UserInterface>(config) || {
+                  'palette.primary.main': theme.palette.primary.main,
+                  'palette.primary.dark': theme.palette.primary.dark,
+                  'palette.primary.light': theme.palette.primary.light,
+                }
```

Comment:
We should merge `config` with the defaults we have. Otherwise, emptying a single field would not benefit from this logic.

Expected result;

- Empty a single field on /admin-settings/user-interface
- Update the configuration by submitting the form
- Expect that single field to have its default value